test/wpt/test-broadcastchannel.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+require('../common');
+const { WPTRunner } = require('../common/wpt');
+
+const runner = new WPTRunner('webmessaging/broadcastchannel');
+
+runner.setInitScript(`
+  const { BroadcastChannel } = require('worker_threads');
+  global.BroadcastChannel = BroadcastChannel;
+  global.location = new URL('https://example.com/');  // Questionable.
```

Comment:
I think we should not run tests that check `global.location` since Node.js doesn't support it.

Reply:
Okay, removed.